ipatests/test_xmlrpc/test_group_plugin.py

```diff
@@ -193,6 +194,27 @@ def test_create_with_numeric_only_group_name(self):
         )):
             testgroup.create()
 
+    def test_rename_to_invalid_groupname(self, group):
+        """ Try to rename group using an invalid name """
+        group.ensure_exists()
+        command = group.make_update_command(
+            updates=dict(setattr=u'cn=%s' % invalidgroup1))
```

Comment:
The proper method to rename a group is to use `ipa group-mod name --rename newname`, which would be translated into
```
        command = group.make_update_command(
            updates=dict(rename=invalidgroup1))
        with raises_exact(errors.ValidationError(
            name='rename',
            error=ERRMSG_GROUPUSER_NAME.format('group'),
        )):
            command()

```

Reply:
Thank you for this suggestion, before I was inspired by test_rename, but this better.

Reply:
@rcritten agreed, but unfortunately using setattr doesn't go through the validation code and lets you use an invalid value. We should open a ticket to fix that.

Reply:
Opened https://pagure.io/freeipa/issue/9396 to track the issue and PR https://github.com/freeipa/freeipa/pull/6883 with a fix + test for the ipa group-mod --setattr cn= part

Reply:
Thank you @flo-renaud for dealing with it.